frontend/components/graphics/FilePkg.tsx

```diff
@@ -1,16 +1,22 @@
 import React from "react";
 
+import { uniqueId } from "lodash";
+
 const FilePkg = () => {
+  const clipPathIdA = uniqueId("clip-path-a-");
+  const clipPathIdB = uniqueId("clip-path-b-");
```

Comment:
Does this need the `b-` prefix? We're gonna get different IDs for each call right?